### ISSUE TITLE
HTML-encoding problem in forms

### ISSUE THREAD

**Issue by djstevenson** [djstevenson/songs-to-the-siren#263]
Create a link that has a description of

```
The Number One Song in Heaven 12" - Sparks (audio of the full 12" version)
```

This saves ok.

But then edit the link, and the form field is pre-filled with just:

```
The Number One Song in Heaven 12
```

Oops.

Probably a blocker to going live, even though this pages are only accessible to the trusted admin.

**Comment by djstevenson** [comment]
Added "improve test coverage" as it clearly needs a test!

**Comment by djstevenson** [comment]
Fixed in PR #264
